pkg/runtime/cli.go

```diff
@@ -363,6 +365,11 @@ func FromFlags() (*DaprRuntime, error) {
 		globalConfig = daprGlobalConfig.LoadDefaultConfiguration()
 	}
 
+	if !daprGlobalConfig.IsFeatureEnabled(globalConfig.Spec.Features, daprGlobalConfig.WorkflowEngine) && *enableWorkflowEngine {
```

Comment:
@ItalyPaleAle your comment is correct about the CLI flag, but it seems to be that AND the feature preview.